### DISCUSSION TITLE
Someone has successfully published with copier template on AWS Lightsail?

### DISCUSSION THREAD

**Discussion by carlosecv** [Tecnativa/doodba-copier-template#462] [General]
I Installed all correctly just at the end My Odoo Portal always says: ERR_SSL_KEY_USAGE_INCOMPATIBLE

**Comment by pleirb** [comment]
> Me instaloba todo correctamente justo al final Mi Portal de Odoo siempre dados: ERR-SSL-KEY-USAGE-INCOMPATIBLE

It sounds to me like Traefik problems with the certificates used for production or test mode
